app/(api)/_schema/collection_schema.mjs

```diff
@@ -0,0 +1,21 @@
+const collection = {
+  bsonType: 'object',
+  title: 'Pokemon Object Validation',
+  properties: {
+    _id: {
+      bsonType: 'objectId',
+      description: '_id must be an ObjectId',
+    },
+    name: {
+      bsonType: 'string',
+      description: 'name must be a string',
+    },
+    happiness: {
+      bsonType: 'number',
+      description: 'happiness must be a number',
```

Comment:
i recently learned that these descriptions lowkey might be useless. Also this is all still pokemon stuff